### PR TITLE
Use props files

### PR DESCRIPTION
**Pull request by KuraiAndras**

### CODE REVIEW

File: Source/Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard2.0|AnyCPU'">
+    <WarningsAsErrors></WarningsAsErrors>
+    <NoWarn>1701;1702</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="..\..\Images\fluxor-small-logo.png">
```
Comment (mrpmorris) [inline Source/Directory.Build.props:38]: This is a different logo per package.

**Comment by mrpmorris** [comment]
Could you base this off release/4.0 please?

**Comment by KuraiAndras** [comment]
I'll solve these today.

**Comment by KuraiAndras** [comment]
Like this?

**Comment by mrpmorris** [comment]
I killed release/4.0 and recreated it.

Would you mind submitting a new PR for this to the new release/4.0 please?

Thanks

**Pull request merged**